src/behaviors/scrollable.js

```diff
@@ -13,11 +13,15 @@ export default function scrollable(WrappedComponent) {
     }
 
     componentDidMount() {
-      this.node.addEventListener('scroll', this.updateScrollState);
+      if (this.node) {
```

Comment:
Doesn't componentDidMount only run once? If `this.node` isn't being found then we have some bigger problems!

That said, this scroll tracking feature isn't required anymore, so perhaps we ought to remove it altogether?